build/pythontool.py

```diff
@@ -1,9 +1,13 @@
 from waflib import Options
-import os, re
+import os, sys, re
 
 def options(opt):
     opt.load('python')
-    opt.add_option('--disable-python', action='store_false', dest='python',
+    if sys.platform == 'sunos5':
+        opt.add_option('--disable-python', action='store_true', 
```

Comment:
Ah, right. Fixed.


From: Adam Sylvester [mailto:notifications@github.com]
Sent: Wednesday, May 04, 2016 12:04 PM
To: mdaus/coda-oss <coda-oss@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>; Author <author@noreply.github.com>
Subject: Re: [mdaus/coda-oss] Fix Solaris build by disabling Python and replacing deprecated os.pop… (#116)


In build/pythontool.py<http://cp.mcafee.com/d/5fHCNEedEIe9IzxPPUUTsSOMOe76zBZYS-epophd79EVdETuKYUqerCzBUSyyOyC-qen67Pozm6P9X3BvS50kGxHtendGEqTjBPqbO2eFzD-LOa8VWWr2fnKnhu79YCCejs7feth5dqWqJTCel3PWApmU6CQjqpK_8K6zBV55BeXNKVIDeqR4IMzapdIxO-aEqTjblgQLmBcmrffN55M349kITiAGM5oK9TssuppdAaJMJZ00Lcfc8IBSm4tbydorcDIel_okelgRKDbwGx4jUzapdIxOVJZyXNI5-Aq83iSGxHtcJl3iZqkdbFEw5V1VEw6aDX6l3h0grAuwq821oQg8iFF0-k29Ew2NqpeI3h1I43h17RzZJa14Qg8uJ7XrfQ3h1mDblmdshG0dkNEwd5068EK6yASzuVKREjYIB3LvV9>:

>

>  def options(opt):

>      opt.load('python')

> -    opt.add_option('--disable-python', action='store_false', dest='python',

> +    if sys.platform == 'sunos5':

> +        opt.add_option('--disable-python', action='store_true',

Actually I misspoke earlier... let's make this one --enable-python. Otherwise they don't have a way to actually turn Python support on in Solaris do they?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/1jWVIi6x8SyMUCOe7ffzztPrb38UsqenTPrUVBxB4QsCzASztWXPxEVKqenzqabaarVEVsovdydorcDIel_ok1iG6JQVsSGxHtendEL88WCevW_8EzDHFI8ZuVt5UsDOqoVdMsYVR4kRHFGTuoVkffGhBrwqrjdFCXYyUqenAkmkXL6XCOsVHkiP2cFASO7bUGxHtcJl3iZqkNpIY_4knctAvZcjd7dSbIcefE6zAs_HB_pd7babH8Uw_H3XW0VwwUD-Ss0ObytT77Cmjp2Hsbvg0bP3P2b9tBx7iUzm6P9X3BvS53BkdrFOUaEh4-8OCjr8sKrvoKYr1vF6y0QJGEqTjblgQLmB3iWq81uguq81yF-NBgQg46V7E6y0wmd424GqgfB0yq80ImCjH0Qgr10QghZo_riwhd427Hh-SPZ0QglFORlzn4qw3lcq83hg1yabxEFdETKrRFJb>


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.